book/src/user/custom-testnets.md

```diff
@@ -0,0 +1,180 @@
+# Custom Testnets
+
+Custom Testnets in Zebra enable testing consensus rule changes on a public, configured Testnet independent of the default public Zcash Testnet.
+
+Zebra's Testnet can be configured with custom:
+- Network upgrade activation heights, 
+- Network names, 
+- Network magics, 
+- Slow start intervals, 
+- Genesis hashes, and
+- Target difficulty limits.
+
+It's also possible to disable Proof-of-Work validation by setting `disable_pow` to `true` so that blocks can be mined onto the chain without valid Equihash solutions, nor block hashes below their target difficulties.
+
+Configuring any of those Testnet parameters except the network name with non-default values will result in an incompatible custom Testnet.
```

Comment:
Can we also clarify what an "incompatible Testnet" means?

Reply:
We mention it a few times later on, so I think it would be useful to have the incompatibility defined at least informally.

Reply:
Described what's meant by "incompatible Testnet" in https://github.com/ZcashFoundation/zebra/pull/8636/commits/0842950c785dca8826127029a9cf26470214dfa5.